src/cai_lab_to_nwb/zaki_2024/interfaces/miniscope_imaging_interface.py

```diff
@@ -205,36 +301,6 @@ def __init__(self, folder_path: DirectoryPath):
 
         self.photon_series_type = "OnePhotonSeries"
 
-    def _get_session_start_time(self):
```

Comment:
Another, equally valid option, is to make these functions static or class methods of the class. This is more of taste and preference and depends on how do you want to organize your code.